tools/go-changelog/cmd/changelog-pr-body-check/main.go

```diff
@@ -57,31 +58,23 @@ func main() {
 		Body:  pullRequest.GetBody(),
 	}
 
-	if err := entry.Validate(); err != nil {
-		log.Printf("error parsing changelog entry in %s: %s", entry.Issue, err)
-		switch err.Code {
-		case changelog.EntryErrorNotFound:
-			body := "Oops! It looks like no changelog entry is attached to" +
-				" this PR. Please include a release note block" +
-				" in the PR body, as described in https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/"
-			log.Fatal(body)
-		case changelog.EntryErrorUnknownTypes:
-			unknownTypes := err.Details["unknownTypes"].([]string)
-
-			body := "Oops! It looks like you're using"
-			if len(unknownTypes) == 1 {
-				body += " an"
-			}
-			body += " unknown release-note type"
-			if len(unknownTypes) > 1 {
-				body += "s"
-			}
-			body += " in your changelog entries:"
-			for _, t := range unknownTypes {
-				body += "\n* " + t
+	if errors := entry.Validate(); errors != nil {
+		log.Printf("error parsing changelog entry in %s: %s", entry.Issue, errors)
```

Comment:
Since each of the errors will get printed out individually below, this output seems like it will just be a (less readable) duplicate... maybe it would be okay to leave out?

Reply:
yeah, this makes sense